app/schemas/models/classroom.stats.schema.js

```diff
@@ -0,0 +1,57 @@
+const c = require('./../schemas')
+
+const ClassroomStatsSchema = c.object({
+  title: 'ClassroomStats',
+  description: 'The stats of a classroom which uses in outcome-reports.',
+})
+
+_.extend(ClassroomStatsSchema.properties, {
+  classroom: c.objectId(),
+  course: c.array({
+    title: 'Course',
+    description: 'The course set of the classroom.',
+    items: c.objectId()
+  }),
+  date: c.stringDate({ description: 'the daily date of the stats' }),
```

Comment:
why are we storing it by date? What if I complete a level on day 1 and for some reason, I play the same level on day 10 in a different language or retry the level?

Reply:
Ok, i see why you store by date because you need stats for a date range.